spec/forms/providers/link_application/find_link_application_form_spec.rb

```diff
@@ -81,6 +81,30 @@
         expect(application_can_be_linked).to be :voided_or_deleted_message
       end
     end
+
+    context "when the application ref searched for has no hyphens" do
+      let(:search_laa_reference) { "L123456" }
+
+      it "returns true" do
+        expect(application_can_be_linked).to be true
+      end
+    end
+
+    context "when the application ref searched for is lower case" do
+      let(:search_laa_reference) { "l-123-456" }
+
+      it "returns true" do
+        expect(application_can_be_linked).to be true
+      end
+    end
+
+    context "when the application ref searched for contains special characters" do
+      let(:search_laa_reference) { "!@L-£$%123-^&*456()" }
```

Comment:
Could we add a space to the test to ensure they get removed too? Just to be explicit!

Reply:
👍 added